styles.css

```diff
@@ -195,28 +184,33 @@ nav {
   border: none;
   width: 18%;
 }
-footer {
-  background-color: #5356A4;
-  display: flex;
-  justify-content: space-between;
-  align-items: stretch;
-  height: 2em;
-  border-top: 1px #1f1f3d solid;
+h3 {
+  font-weight: bold;
+  color: #1F1F3D;
 }
-.comment-button {
-  background: url(assets/comment.svg);
-  background-repeat: no-repeat;
-  width: 18%;
+p {
```

Comment:
It's a good idea to target things by class name instead of tag because if you add <p> tags later, they will all inherit this styling (which you may not want)